src/components/moderation/ReportDialog/utils/useReportOptions.ts

```diff
@@ -1,121 +1,295 @@
+      description: _(
+        msg`Content harming or endangering minors' safety and wellbeing`,
+      ),
+      sort: 1,
+      options: [
+        {
+          title: _(msg`Child Sexual Abuse Material (CSAM)`),
+          reason: OzoneReportDefs.REASONCHILDSAFETYCSAM,
+        },
+        {
+          title: _(msg`Grooming or Predatory Behavior`),
+          reason: OzoneReportDefs.REASONCHILDSAFETYGROOM,
+        },
+        {
+          title: _(msg`Minor Privacy Violation`),
+          reason: OzoneReportDefs.REASONCHILDSAFETYMINORPRIVACY,
+        },
+        {
+          title: _(msg`Child Endangerment`),
+          reason: OzoneReportDefs.REASONCHILDSAFETYENDANGERMENT,
+        },
+        {
+          title: _(msg`Minor Harassment or Bullying`),
+          reason: OzoneReportDefs.REASONCHILDSAFETYHARASSMENT,
+        },
+        {
+          title: _(msg`Promotion of Child Exploitation`),
+          reason: OzoneReportDefs.REASONCHILDSAFETYPROMOTION,
+        },
+        {
+          title: _(msg`Other Child Safety Issue`),
+          reason: OzoneReportDefs.REASONCHILDSAFETYOTHER,
+        },
       ],
-      post: [
+    },
+    violencePhysicalHarm: {
+      key: 'violencePhysicalHarm',
+      title: _(msg`Violence or Physical Harm`),
+      sort: 2,
+      description: _(msg`Threats, calls for violence, or graphic content`),
+      options: [
         {
-          reason: ComAtprotoModerationDefs.REASONMISLEADING,
-          title: _(msg`Misleading Post`),
-          description: _(msg`Impersonation, misinformation, or false claims`),
+          title: _(msg`Animal Welfare`),
+          reason: OzoneReportDefs.REASONVIOLENCEANIMALWELFARE,
         },
         {
-          reason: ComAtprotoModerationDefs.REASONSPAM,
-          title: _(msg`Spam`),
-          description: _(msg`Excessive mentions or replies`),
+          title: _(msg`Threats or Incitement`),
+          reason: OzoneReportDefs.REASONVIOLENCETHREATS,
+        },
+        {
+          title: _(msg`Graphic Violent Content`),
+          reason: OzoneReportDefs.REASONVIOLENCEGRAPHICCONTENT,
+        },
+        {
+          title: _(msg`Self Harm`),
+          reason: OzoneReportDefs.REASONVIOLENCESELFHARM,
+        },
+        {
+          title: _(msg`Glorification of Violence`),
+          reason: OzoneReportDefs.REASONVIOLENCEGLORIFICATION,
+        },
+        {
+          title: _(msg`Extremist Content`),
+          reason: OzoneReportDefs.REASONVIOLENCEEXTREMISTCONTENT,
+        },
+        {
+          title: _(msg`Human Trafficking`),
+          reason: OzoneReportDefs.REASONVIOLENCETRAFFICKING,
+        },
+        {
+          title: _(msg`Other Violent Content`),
+          reason: OzoneReportDefs.REASONVIOLENCEOTHER,
+        },
+      ],
+    },
+    sexualAdultContent: {
+      key: 'sexualAdultContent',
+      title: _(msg`Sexual and Adult Content`),
+      description: _(msg`Adult, child or animal sexual abuse`),
+      sort: 3,
+      options: [
```

Comment:
```suggestion
      options: [
        {
          title: _(msg`Unlabeled Adult Content`),
          reason: OzoneReportDefs.REASONSEXUALUNLABELED,
        },
```
Suggest moving `Unlabeled Adult Content` to be the first option to make it easier for users to find, as I suspect it's very likely to be the most-used option in this category.